### PR TITLE
#310: Create stub files for all exercises - difference-of-square…

### PR DESCRIPTION
**Pull request by eparovyshnaya**

### CODE REVIEW

**Comment by uzilan** [comment]
LGFM :)

File: exercises/difference-of-squares/src/main/kotlin/Squares.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Squares {
```
Comment (dector) [inline exercises/difference-of-squares/src/main/kotlin/Squares.kt]: Do we really need class here?

Comment (eparovyshnaya) [reply exercises/difference-of-squares/src/main/kotlin/Squares.kt]: I'm kinda guided my the appropriate test suite. 

We might be willing to shape a general convension and to rework all the tests suits/ref impls/stubs bundles according to it. But this all is out of the scope of this PR and the issue is is dedicated to. I can post a proper issue for further review/rework, but freeze here at what existing test suit requires. Do you agree?

Comment (dector) [reply exercises/difference-of-squares/src/main/kotlin/Squares.kt]: Awesome! You are totally right.

Comment (eparovyshnaya) [reply exercises/difference-of-squares/src/main/kotlin/Squares.kt]: @dector, issue #349

**Pull request merged**